roles/test_operator/defaults/main.yml

```diff
@@ -62,7 +62,7 @@ cifmw_test_operator_tempest_config:
         {{ cifmw_test_operator_tempest_exclude_list | default('') }}
       concurrency: "{{ cifmw_test_operator_concurrency }}"
       externalPlugin: "{{ cifmw_test_operator_tempest_external_plugin | default([]) }}"
-      neutronExtraImage: "{{ cifmw_test_operator_tempest_ntp_extra_image | default('') }}"
+      extraImage: "{{ cifmw_test_operator_tempest_extra_images | default([]) }}"
```

Comment:
see about about having this as extraImages

Reply:
Good point, thank you!